# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../transcode/fix-autoconf-failure.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

===================================================================
RCS file: /cvstc/transcode/acinclude.m4,v
retrieving revision 1.72
retrieving revision 1.73
diff -u -r1.72 -r1.73
--- transcode/acinclude.m4	2006/06/06 02:10:52	1.72
+++ transcode/acinclude.m4	2006/07/29 09:05:30	1.73
@@ -191,7 +191,7 @@
 ],    [AC_DEFINE([HAVE_STRUCT_V4L2_BUFFER], 1,
         [define if your videodev2 header has struct v4l2_buffer])
         AC_MSG_RESULT([yes])],
-      [v4l2=no AC_MSG_RESULT([no])])
+      [AC_MSG_RESULT([no])])
   fi
 
   if test x"$v4l" = x"yes" -o x"$v4l2" = x"yes" ; then
